src/docs/README.adoc

```diff
@@ -91,27 +87,19 @@ image::https://lucid.app/publicSegments/view/2cb3b7e2-bfdf-4e78-8247-22ec394de96
 === Background
 
 The core Kafka consumer client gives you a batch of messages to process one at a time.
-Processing these in parallel on thread pools is difficult, particularly due to ensuring correctness of offset management, especially when trying to process concurrent by message key.
-
-You also need to manage your consume loop, and commit transactions properly if using Exactly Once semantics.
+Processing these in parallel on thread pools is difficult, particularly when considering offset management and strong ordering guarentees. You also need to manage your consume loop, and commit transactions properly if using Exactly Once semantics.
 
-This wrapper library for the core Kafka client handles all this for you, you just supply your processing function.
+This wrapper library for the Apache Kafka Java client handles all this for you, you just supply your processing function.
 
-A very common example of a scenario where this helps tremulously, is in "slow consumer" situations, where processing a single message in your topic is slow, but the messages aren't so dependent on each other.Slow processing can be typically CPU bound, IO bound or bound by third party.The VertX extension to this library supplies many non-blocking interfaces allowing you to achieve high levels of concurrency easily.
+A very common example of a scenario where this helps tremulously, is in "slow consumer" situations, where processing a single message in your topic is slow, but the messages aren't so dependent on each other. Slow processing can be typically CPU bound, IO bound or bound by third party. The VertX extension to this library supplies many non-blocking interfaces allowing you to achieve high levels of concurrency easily.
```

Comment:
@astubbs I don't really follow this use case. Are you trying to get at the queuing feature here?